Gaming The Syllabus/Assets/Scripts/Tooltip.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class Tooltip : MonoBehaviour, IPointerEnterHandler, IPointerExitHandler
```

Comment:
Mega-props for making this general use so we can use it for anything. That's absolutely awesome